ibm_db.c

```diff
@@ -17174,10 +17071,15 @@ static PyObject *ibm_db_fetchall(PyObject *self, PyObject *args)
                 Py_XDECREF(result_list);
```

Comment:
return_value is not getting free if control comes inside this `if` block. Free it just before while loop.